XRTK-Core/Packages/com.xrtk.core/Providers/Controllers/BaseController.cs

```diff
@@ -213,11 +213,10 @@ internal async Task TryRenderControllerModelAsync(Type controllerType, byte[] gl
 
             if (!visualizationProfile.RenderMotionControllers) { return; }
 
-            GameObject controllerModel = null;
             GltfObject gltfObject = null;
 
             // If a specific controller template exists, check if it wants to override the global model, or use the system default specifically (in case global default is not used)
-            bool useSystemDefaultModels = visualizationProfile.GetControllerModelOverride(controllerType, ControllerHandedness, out controllerModel);
+            bool useSystemDefaultModels = visualizationProfile.GetControllerModelOverride(controllerType, ControllerHandedness, out var controllerModel);
```

Comment:
Not a fan of var here but I guess that's personal preference.

Reply:
Just a product of in-lining the out variable. I don't really mind, more concise.